sdk/parents/azure-client-sdk-parent/pom.xml

```diff
@@ -469,6 +388,112 @@
+            </dependency>
+          </dependencies>
+          <configuration>
+            <configLocation>../../../eng/code-quality-reports/src/main/resources/checkstyle/checkstyle.xml</configLocation>
+            <suppressionsLocation>../../../eng/code-quality-reports/src/main/resources/checkstyle/checkstyle-suppressions.xml</suppressionsLocation>
+            <headerLocation>../../../eng/code-quality-reports/src/main/resources/checkstyle/java.header</headerLocation>
+            <propertyExpansion>samedir=</propertyExpansion>
+            <encoding>UTF-8</encoding>
+            <consoleOutput>true</consoleOutput>
+            <includeTestSourceDirectory>true</includeTestSourceDirectory>
+            <linkXRef>true</linkXRef>
+            <failsOnError>true</failsOnError>
+            <failOnViolation>true</failOnViolation>
+          </configuration>
+        </plugin>
+
+        <!-- This plugin scans reports spotbugs in the code -->
+        <plugin>
+          <groupId>com.github.spotbugs</groupId>
+          <artifactId>spotbugs-maven-plugin</artifactId>
+          <version>3.1.12.2</version> <!-- {x-version-update;com.github.spotbugs:spotbugs-maven-plugin;external_dependency} -->
+          <dependencies>
+            <dependency>
+              <groupId>com.azure</groupId>
+              <artifactId>sdk-build-tools</artifactId>
+              <version>1.0.0</version> <!-- {x-version-update;com.azure:sdk-build-tools;external_dependency} -->
+            </dependency>
+            <dependency>
+              <groupId>com.github.spotbugs</groupId>
+              <artifactId>spotbugs</artifactId>
+              <version>4.0.0-beta3</version> <!-- {x-version-update;com.github.spotbugs:spotbugs;external_dependency} -->
+            </dependency>
+          </dependencies>
+          <configuration>
+            <effort>max</effort>
+            <threshold>Low</threshold>
+            <xmlOutput>true</xmlOutput>
+            <spotbugsXmlOutputDirectory>${project.build.directory}/spotbugs</spotbugsXmlOutputDirectory>
+            <excludeFilterFile>spotbugs/spotbugs-exclude.xml</excludeFilterFile>
+            <fork>true</fork>
+            <failOnError>true</failOnError>
+            <!-- Set this to true when src/samples and src/test are spotbugs clean in all Track 2 SDKs -->
+            <includeTests>false</includeTests>
+
+            <!-- Enable this plugin when issues documented are fixed -->
+            <!--          <plugins>-->
+            <!--            <plugin>-->
+            <!--              <groupId>com.h3xstream.findsecbugs</groupId>-->
+            <!--              <artifactId>findsecbugs-plugin</artifactId>-->
+            <!--              <version>1.9.0</version> &lt;!&ndash; {x-version-update;com.h3xstream.findsecbugs:findsecbugs-plugin;external_dependency} &ndash;&gt;-->
+            <!--            </plugin>-->
+            <!--          </plugins>-->
```

Comment:
What are the issues that need to be fixed to re-enable this? This seems to be commented out for a long time. Should we re-enable this and see what issues crop up and file issues to track them?

Reply:
@g2vinay do you know if these have been resolved?

I can file a follow-up issue to enable this.